[2022-06-11 17:07:39,595] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: T-dbt-job-dimensions.csv_to_database manual__2022-06-11T17:07:37.200111+00:00 [queued]>
[2022-06-11 17:07:39,606] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: T-dbt-job-dimensions.csv_to_database manual__2022-06-11T17:07:37.200111+00:00 [queued]>
[2022-06-11 17:07:39,606] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-11 17:07:39,607] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-06-11 17:07:39,607] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-11 17:07:39,621] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): csv_to_database> on 2022-06-11 17:07:37.200111+00:00
[2022-06-11 17:07:39,634] {standard_task_runner.py:52} INFO - Started process 540 to run task
[2022-06-11 17:07:39,638] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'T-dbt-job-dimensions', 'csv_to_database', 'manual__2022-06-11T17:07:37.200111+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/docker_job/dbt_test.py', '--cfg-path', '/tmp/tmpicnqm7oi', '--error-file', '/tmp/tmpb8ovjgto']
[2022-06-11 17:07:39,638] {standard_task_runner.py:80} INFO - Job 40: Subtask csv_to_database
[2022-06-11 17:07:39,704] {task_command.py:370} INFO - Running <TaskInstance: T-dbt-job-dimensions.csv_to_database manual__2022-06-11T17:07:37.200111+00:00 [running]> on host d8836eb746ac
[2022-06-11 17:07:39,788] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=samuelolle@yahoo.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=T-dbt-job-dimensions
AIRFLOW_CTX_TASK_ID=csv_to_database
AIRFLOW_CTX_EXECUTION_DATE=2022-06-11T17:07:37.200111+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-11T17:07:37.200111+00:00
[2022-06-11 17:07:39,797] {base.py:68} INFO - Using connection ID '***' for task execution.
[2022-06-11 17:07:39,801] {postgres.py:127} INFO - Running copy expert: COPY analytics_object.users_cdc FROM STDIN, filename: /opt/airflow/dags/docker_job/data/users.csv
[2022-06-11 17:07:39,809] {base.py:68} INFO - Using connection ID '***' for task execution.
[2022-06-11 17:07:39,822] {base.py:68} INFO - Using connection ID '***' for task execution.
[2022-06-11 17:07:39,826] {postgres.py:127} INFO - Running copy expert: COPY analytics_object.walls_cdc FROM STDIN, filename: /opt/airflow/dags/docker_job/data/walls.csv
[2022-06-11 17:07:39,834] {base.py:68} INFO - Using connection ID '***' for task execution.
[2022-06-11 17:07:39,841] {python.py:173} INFO - Done. Returned value was: None
[2022-06-11 17:07:39,857] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=T-dbt-job-dimensions, task_id=csv_to_database, execution_date=20220611T170737, start_date=20220611T170739, end_date=20220611T170739
[2022-06-11 17:07:39,891] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-11 17:07:39,931] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
